### ISSUE TITLE
Breaker list shows breakers that don't exist anymore

### ISSUE THREAD

**Issue by poisa** [MarkBryanMilligan/LanternPowerMonitor#36]
I've renamed some breakers, but the old ones "sometimes" linger in my list of breakers (I haven't been able to figure out when this happens). For example, I have an old one called "Space 7" which I renamed to simply "7" but both appear in the list.
I've reset both hubs several times since I've done this, but the problem persists.

**Comment by MarkBryanMilligan** [comment]
I think this bug is in the iOS app, specifically when switching from single pole to double pole, one CT.  For now I cleaned it up in the database.  I'll put this on my todo list for when I go in and add the billing rate config to iOS.

**Comment by poisa** [comment]
> when switching from single pole to double pole, one CT

Yeah, that's exactly what happened in my case. Thanks for the prompt response. Looking forward to the new iOS version!
